schedule-additional-tests

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -ex
 CONFIG_MAP=$(dirname $0)/config.map
-PROFILING_WORKFLOWS=$(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||' | tr ',' ' ')
+if [ "X$PROFILING_WORKFLOWS" == "X" ];then
+  PROFILING_WORKFLOWS=$(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||' | tr ',' ' ')
+fi
```

Comment:
this is only used for IBs test. So I do not think we need this change. For IBs we should get workflows from the cmssw-pr-test-config file. You need update https://github.com/cms-sw/cms-bot/blob/master/pr-schedule-tests#L102 and add a line 
```
echo "PROFILING_WORKFLOWS=${PROFILING_WORKFLOWS}" >> $OUTPUT_FILE
```